client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -258,7 +261,7 @@ export const AppSchemaForm = ({ app }) => {
 
   return (
     <div id="appForm-wrapper">
-      {!systemHasKeys && (
+      {!!(!systemHasKeys && hasStorageSystems) && (
```

Comment:
Curious about the use of double exclamation marks - as I understand it here, any variables cast with a `&&` will result in a boolean. What is your impetus to use them here?

Reply:
It displayed a `0` for two comparisons of three values (i.e. omitting the double exclamation). It seems like this shorthand only works for two comparisons.
So, I tried to use a ternary operator to ensure that there is only `&&` comparison and it worked.
Then, eslint just auto-fixed it to be a double exclamation for simplicity.
I think the double exclamation forces the former comparison to be an explicit single boolean value to make this shorthand work.

Reply:
Interesting, thanks!

Reply:
@duckonomy Should we add a small comment somewhere explaining this?